src/topology/basic.lean

```diff
@@ -426,22 +426,28 @@ end
 ### Neighborhoods
 -/
 
-/-- neighbourhood filter -/
+/-- The neighborhoods of a point `a` are the sets which include an open set containing `a`. These
```

Comment:
```suggestion
/-- The neighborhoods of a point `a` are the sets which contain an open set containing `a`. These
```

Reply:
"Contain" would incorrectly imply that `open set ∈ neighborhood`, so I am hesitant to use that.

Would you prefer

```lean
/-- The neighborhoods of a point `a` are the sets whose interior contains `a`. These sets form a
filter. -/
```

instead?

Reply:
A set is called a neighborhood of `a` if it contains an open set around `a`. The set of all neghborhoods of `a` forms a filter, the neighorhood filter at `a`, which is here defined as the infimum over the principal filters of all open sets containing `a`.